### ISSUE TITLE
Potential issues with .deb packages

### ISSUE THREAD

**Issue by ronaldodorneles** [autobrr/autobrr#1599]
**Version**
The autobrr version you're running. If you run something other than latest, please update before reporting issues.

**Describe the bug**
I use ubuntu, 24.04 and installed 1.42.deb
I access by SSH and stop the service systemctl and install deb by comand sudo dpkg -i autobrr_1.44.0_linux_amd64.deb
then terminal prints :
(Reading database ... 201557 files and directories currently installed.)
Preparing to unpack autobrr_1.44.0_linux_amd64.deb ...
Unpacking autobrr (1.44.0) over (1.43.0) ...
Setting up autobrr (1.44.0) ...

Then i start systemctl service , enter in 

![Screenshot from 2024-07-20 16-03-01](https://github.com/user-attachments/assets/dbaf619e-8ab9-4704-a7b1-4c8b1daa679e)

**Comment by ronaldodorneles** [comment]
> Can you explain what's wrong? does autobrr not start?

Sorry!
Start version 1.42.

**Comment by ronaldodorneles** [comment]
> Can you explain what's wrong? does autobrr not start?

Starts on version 1.42, don't starts on 1.44.

**Comment by zze0s** [comment]
Can you share your systemd unit file @ronaldodorneles ?

I just tested in a clean ubuntu 24.04 vm and it works as intended. Started with 1.42.deb:

I used the systemd unit file we have [on the docs here](https://autobrr.com/installation/linux#systemd-recommended) but changed the binary path from `/usr/local/bin/autobrr` to `/usr/bin/autobrr` since that is where install via the .deb package seem to put it.

Using that then `sudo systemctl enable --now autobrr@$USER.service` and a status check `sudo systemctl status autobrr@$USER.service` and it runs as expected.

Then `sudo dpkg -i autobrr_1.44.0_linux_amd64.deb` and a restart of the service `sudo systemctl restart autobrr@$USER.service` and it starts as expected with the new version.

TLDR: Works on a clean install so it must be something with your environment.

**Comment by ronaldodorneles** [comment]
> Can you share your systemd unit file @ronaldodorneles ?
> 
> I just tested in a clean ubuntu 24.04 vm and it works as intended. Started with 1.42.deb:
> 
> I used the systemd unit file we have [on the docs here](https://autobrr.com/installation/linux#systemd-recommended) but changed the binary path from `/usr/local/bin/autobrr` to `/usr/bin/autobrr` since that is where install via the .deb package seem to put it.
> 
> Using that then `sudo systemctl enable --now autobrr@$USER.service` and a status check `sudo systemctl status autobrr@$USER.service` and it runs as expected.
> 
> Then `sudo dpkg -i autobrr_1.44.0_linux_amd64.deb` and a restart of the service `sudo systemctl restart autobrr@$USER.service` and it starts as expected with the new version.
> 
> TLDR: Works on a clean install so it must be something with your environment.

IT Works!
Thanks, i changed the binary path from /usr/bin/autobrr and worked !
